### DISCUSSION TITLE
新功能：笔记增加私密

### DISCUSSION THREAD

**Discussion by zwmmm** [tw93/MiaoYan#247] [Ideas]
类似于熊掌记这样的，私密笔记单独上锁

使用场景就是我会在上面记录一些敏感信息，比如Github的Token，邮箱的Token，个人简历等

**Comment by tw93** [comment]
这个诉求挺好的，我想想哈
不过我认为电脑离开不少不是应该会关上或者锁屏吗

**Comment by zwmmm** [comment]
总归还是有的时候电脑屏幕有人看着

**Comment by tw93** [comment]
也即类似于“防君子不防小人”那种意思不，因为我看假如需要彻底弄加密这一块，其实过于复杂了，假如简单实现这个功能，应该还行

**Comment by Cspdhsg** [comment]
老哥可以参考onenote笔记的加密功能，实现起来应该也不复杂 @tw93

**Comment by whsheng** [comment]
可以通过某个开源的加密应用，通过命令行方式对指定的文件加解密处理 @tw93 @zwmmm
